src/stream/src/executor/mview/materialize.rs

```diff
@@ -64,6 +64,10 @@ pub struct MaterializeExecutor<S: StateStore, SD: ValueRowSerde> {
     materialize_cache: MaterializeCache<SD>,
 
     conflict_behavior: ConflictBehavior,
+
+    metrics: Arc<StreamingMetrics>,
+
+    table_id: u32,
```

Comment:
ditto. we can access `table_id` from https://github.com/risingwavelabs/risingwave/blob/ca18ca82fca480202c4d5dfed2af529d1efcf5ca/src/stream/src/executor/mview/materialize.rs#L55